### ISSUE TITLE
Impossible to specify module/class to `python_client`

### ISSUE THREAD

**Issue by diwakergupta** [dropbox/stone#3]
I'm having trouble running `stone python_client`: it seems to require `--module-name` and `--class-name`, but it seems impossible to actually specify them:

```bash
$ stone python_client python dropbox-api-spec/*.stone
usage: python-client-generator [-h] -m MODULE_NAME -c CLASS_NAME
python-client-generator: error: the following arguments are required: -m/--module-name, -c/--class-name

$ stone --module-name dbx --class-name Api python_client python dropbox-api-spec/*.stone
usage: stone [-h] [-v] [--clean-build] [-f FILTER_BY_ROUTE_ATTR]
             [-a ATTRIBUTE]
             [-w WHITELIST_NAMESPACE_ROUTES | -b BLACKLIST_NAMESPACE_ROUTES]
             generator output [spec [spec ...]]
stone: error: unrecognized arguments: --module-name --class-name

# I tried all possible places for specifying the arguments: after all positional args, after python_client etc
# Always fails regardless
```

**Comment by braincore** [comment]
Stone arguments are separated from generator-specific arguments by a `--`. A valid call would look like:

```
$ stone python_client python dropbox-api-spec/*.stone -- -m base -c Api
```

I'll leave the task open until I improve the documentation.

**Comment by braincore** [comment]
Fixed in b60ab9d6385616c8b00ef31901244ff546ac7562. Thanks!
